packages/jest-haste-map/src/index.ts

```diff
@@ -727,6 +732,41 @@ class HasteMap extends EventEmitter {
       roots: options.roots,
     };
 
+    const watchmanConfigPath = path.join(options.rootDir, '.watchmanconfig');
+    if (existsSync(watchmanConfigPath)) {
+      try {
+        const configStr = readFileSync(watchmanConfigPath).toString().trim();
```

Comment:
```suggestion
        const configStr = readFileSync(watchmanConfigPath, 'utf8').trim();
```

Reply:
why was this suggestion not applied? It does the same thing under the hood, but is more explicit IMO

https://github.com/nodejs/node/blob/3e2a3007107b7a100794f4e4adbde19263fc7464/lib/fs.js#L408

Reply:
Still this 😃

Reply:
Sorry, just merge conflict got in the way and I didnt' notice